### PR TITLE
follows #101 and #105

### PR DESCRIPTION
**Pull request by antograssiot**

### CODE REVIEW

File: tests/TestCase/Shell/BakeShellTest.php [hunk 1]
```diff
@@ -106,7 +106,7 @@ public function testMain()
             ->method('out')
             ->with($this->stringContains('The following commands'));
 
-        $this->Shell->expects($this->exactly(17))
+        $this->Shell->expects($this->exactly(18))
```
Comment (AD7six) [inline tests/TestCase/Shell/BakeShellTest.php]: I think it's about time this test was changed to be actually useful "this method calls out 18 times" is of little to no value.

Comment (markstory) [inline tests/TestCase/Shell/BakeShellTest.php]: I agree this is getting to be diminishing returns at this point.

Comment (AD7six) [inline tests/TestCase/Shell/BakeShellTest.php]: Addessed (hopefully) in https://github.com/cakephp/bake/pull/107

**Comment by antograssiot** [comment]
closing, I'll reopen an other one

**Pull request closed without merging**